playground/frontend/lib/config/theme.dart

```diff
@@ -198,8 +291,9 @@ class ThemeColors {
   Color get secondaryBackground =>
       isDark ? kDarkSecondaryBackground : kLightSecondaryBackground;
 
-  Color get primaryBackground =>
-      isDark ? kDarkPrimaryBackground : kLightPrimaryBackground;
+  Color get background =>
```

Comment:
To match Flutter's `ThemeData.background` that we populate with the same constants.